spacecmd/spacecmd.changes

```diff
@@ -2,6 +2,7 @@
 - Add unit test for schedule, errata, user, utils, misc, configchannel
   and kickstart modules
 - Multiple minor bugfixes alongside the unit tests
+- Add spacecmd missing runtime dependencies: python3-rpm, python3-simplejson (bsc#1148311)
```

Comment:
I'd suggest making this entry more relevant for users/customers as it doesn't briefly explain what was broken.

Reply:
Done